src/components/calcite-slider/calcite-slider.tsx

```diff
@@ -52,6 +52,12 @@ export class CalciteSlider {
     this.hasHistogram = !!newHistogram;
   }
 
+  /**
+   * Array of values describing a single color stop ([offset, color, opacity]).
```

Comment:
@jcfranco sorry just seeing this! Yes let's omit that text. Since it's a non-inlined custom interface type, Stencil docs it as `"type": "ColorStop[]"`.

To articulate the contents for these custom types, I've been adding `@see` JSDoc tags with a markdown link that leads to the interface definition on GH once in master. (I can do that later, because I still have to create a new issue to "doc undocumented props/methods", and review the docs-json fixes we need for dev site with @macandcheese.)